### ISSUE TITLE
compatible with django 3?

### ISSUE THREAD

**Issue by nh916** [dstufft/django-passwords#58]
is this package compatible with django 3?
Also in the future how can I know this without having to ask?
thank you!

**Comment by maccesch** [comment]
yeah good questions. I actually don't know the answer to both. At the moment no one is really maintaining this

**Comment by nh916** [comment]
is there a way we can find out? 
or is there another package that does the same thing?

**Comment by maccesch** [comment]
I think django has sth like this built in now. It may be a bit less powerful but probably enough for most use cases

**Comment by darakian** [comment]
@maccesch If there are plans to deprecate this library could you formalize them? Also can you link to the django built in replacement?

**Comment by mecampbellsoup** [comment]
> I think django has sth like this built in now. It may be a bit less powerful but probably enough for most use cases

What is it?

**Comment by dhararon** [comment]
Maybe this can be useful. https://github.com/Code-Lovers-Club/django-password-validator
